src/include/sof/audio/dts/DtsSofInterfaceVersion.h

```diff
@@ -8,10 +8,10 @@
 
 #define DTS_SOF_INTERFACE_PRODUCT_NAME "DtsSofInterface"
 #define DTS_SOF_INTERFACE_PRODUCT_VERSION_MAJOR 1
-#define DTS_SOF_INTERFACE_PRODUCT_VERSION_MINOR 0
-#define DTS_SOF_INTERFACE_PRODUCT_VERSION_PATCH 7
+#define DTS_SOF_INTERFACE_PRODUCT_VERSION_MINOR 1
```

Comment:
This commit could merge into [dts : add require changes for MTL](https://github.com/joechengxperi/sof/pull/1/commits/9f81c055860c90f243f5f933b304b4ed1ee538be)
Because it takes effect when commit dts : add require changes for MTL merge.